### ISSUE TITLE
Intermittent 400 error on ballot POST endpoint

### ISSUE THREAD

**Issue by MorganLove** [votingworks/arlo#229]
This is something that has cropped up on rare occasions before, and we thought it was just an artifact of unrealistic data, but it has started cropping up more consistently (specifically on the votingworks/arlo#198 PR, which is causing it to fail CI). When using the exact same data (ballot numbers, the POST to the ballot endpoint in the data entry flow is resulting in a 400 server error, causing the end to end tests to fail. Sometimes it will do it on the first ballot, sometimes on the 8th, or other ones. It is not consistent.

**Comment by MorganLove** [comment]
I was able to capture the full error message:

```
errorType: "BadRequest"
message: "Multiple ballots found with election_id=34ad2a46-4c18-43ae-a073-2a13f7ccc232, jurisdiction_id=1442c5f6-4cbf-4d98-bd05-8d6bad5f8ee7, batch_id=b650702a-031a-4f3d-8a0d-89c5f63e1706, ballot_position=176, round=900721f6-f458-4d1f-b9db-6b83fb4ed374"
```

Is this possibly because of the new ticket dimension to the data model, requiring further specificity in the endpoint? If so, this could be a different problem than the one that was popping up rarely before.

**Comment by MorganLove** [comment]
Yes, the other bug seems to be different: https://github.com/votingworks/arlo/pull/119#issuecomment-557322366

**Comment by benadida** [comment]
Good catch @MorganLove so this is the issue where there is now more than one `SampledBallot` row per ballot, there is one per sample, and so we have to decide where we stash the vote itself. Either we normalize the `vote` out to another table, or we move `SampledBallot` back to one row per ballot, with a field that lists all the ticket numbers.

**Comment by benadida** [comment]
will be solved by votingworks/arlo#252
